fix: 統合テスト crawl-cli.test.ts の full.md アサーション失敗を修正

## 概要

統合テスト `tests/integration/crawl-cli.test.ts` の `should have valid full.md content` テストが失敗している。

## 変更対象ファイル
- `link-crawler/tests/integration/crawl-cli.test.ts` (1箇所)

## 問題

テスト（254行目）が `full.md` に `url: https://example.com` (frontmatter形式) が含まれることをアサートしているが、`full.md` の実際のフォーマットは `> Source: https://example.com` 形式。

`Merger.buildFullContent()` は frontmatter なしで `# タイトル\n\n> Source: URL` のフォーマットで結合する。テストが古い仕様を期待しており、現在の実装と一致していない。

**エラー出力:**
```
AssertionError: expected # Example Domain\n\n> Source: https://… to contain url: https://example.com
```

## 影響
- `bun run test` が失敗する (exit code 1)
- CIが通らない

## 修正内容

| ファイル | 行 | Before | After |
|----------|-----|--------|-------|
| `tests/integration/crawl-cli.test.ts` | 254 | `expect(fullContent).toContain("url: https://example.com");` | `expect(fullContent).toContain("> Source: https://example.com");` |

## 検証方法
```bash
cd link-crawler && bun run test tests/integration/crawl-cli.test.ts
# 期待: 全テストパス
```

## 完了条件
- [ ] 上記1箇所を修正
- [ ] `bun run test` が全テストパス（856テスト）

--- a/link-crawler/tests/integration/crawl-cli.test.ts
+++ b/link-crawler/tests/integration/crawl-cli.test.ts
@@ -251,7 +251,7 @@ describe("crawl CLI integration", () => {
 
 			const fullContent = readFileSync(join(outputDir, "full.md"), "utf-8");
 			expect(fullContent).toContain("# ");
-			expect(fullContent).toContain("url: https://example.com");
+			expect(fullContent).toContain("> Source: https://example.com");
 		});
 
 		it("should have valid pages directory content", () => {

